pkg/render/applicationlayer/applicationlayer.go

```diff
@@ -130,11 +161,6 @@ func (c *component) Objects() ([]client.Object, []client.Object) {
 		objs = append(objs, c.securityContextConstraints())
 	}
 
-	// Delete all the objects if logs are not enabled.
```

Comment:
We still might have only one of logs / WAF enabled - if this is the case, we should delete any resources that aren't necessary for the operation of the other.

e.g., if logging is enabled but WAF is disabled, we probably should delete the modsec-ruleset from the namespace. 

This isn't critical though and we can address this in a follow-on - just create a backlog item for it.

Reply:
Will do, thanks!